build/configs/esp32_DevKitC/hello_with_tash/defconfig

```diff
@@ -156,7 +156,7 @@ CONFIG_ARCH_BOARD="esp32_devkit"
 # CONFIG_BOARD_CRASHDUMP is not set
 # CONFIG_LIB_BOARDCTL is not set
 # CONFIG_BOARD_FOTA_SUPPORT is not set
-
+CONFIG_ESP32_WIFI_SUPPORT=y
```

Comment:
is this file generated by menuconfig? or you add new CONFIG_* into existed file manually?
the former is suggested, otherwise, it's inconvenient to use menuconfig to modify configration.

Reply:
I mean, it's better to edit Kconfig files firstly and then make menuconfig to generate `.config`, at last, rename `.config` to `defconfig` and replace the old one.